src/Compilers/Core/MSBuildTask/CanonicalError.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
Can we enable nullability at the project-level instead?

Reply:
This would break tools used to evaluate the status of the Roslyn migration as a whole.

Reply:
Consider doing both (`#nullable enable` in files, and also `Nullable=enable` in project). Enabling at the project level ensures that we will not regress as we add new files.

Reply:
@jaredpar any objection to this plan?

Reply:
Now implemented